common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -61,6 +68,23 @@ public static void logMessage(Level level, MessageAuditLogEntry.Builder entryBui
             .setComponent(buildProperties.getArtifact())
             .setVersion(buildProperties.getVersion())
             .build());
+    Map<String, Object> fluentDLogs = new HashMap<>();
```

Comment:
Nitpick, but I think I prefer `fluentdLogs` so that we dont have two capital letters after each other.

Reply:
Updated.